missing/setproctitle.c

```diff
@@ -135,6 +135,16 @@ compat_init_setproctitle(int argc, char *argv[])
 }
 
 #ifndef HAVE_SETPROCTITLE
+
+void
+rb_free_environ(void) {
+    size_t i;
+    char **envp = environ;
+    for (i = 0; envp[i] != NULL; i++)
+        free(environ[i]);
+    free(environ);
+}
```

Comment:
- `environ` is copied conditionally.
- Why not using `envp` inside the loop?
- Adjust the style.

```suggestion
rb_free_environ(void)
{
#if SPT_TYPE == SPT_REUSEARGV
	size_t i;
	char **envp = environ;
	for (i = 0; envp[i] != NULL; i++)
		free(envp[i]);
	free(environ);
#endif
}
```